src/ctia/http/server.clj

```diff
@@ -185,6 +186,9 @@
             (when-let [lifetime (:lifetime-in-sec jwt)]
               {:jwt-max-lifetime-in-sec lifetime}))))
 
+         (:enabled jwt)
```

Comment:
I think this should always be `true` here.
Or perhaps should be configurable to be on or off via a configuration. But this should not be linked to the `jwt` property.

Reply:
Fixed and added a `ctia.log.riemann.enabled` property.